src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncNodesRequestHandler.scala

```diff
@@ -77,7 +77,7 @@ class FastSyncNodesRequestHandler(
       val evm = n.getAccount.codeHash
       val storage = n.getAccount.storageRoot
 
-      nodesKeyValueStorage.put(n.hash, n.toBytes)
+      saveNodeFn(n.hash, n.toBytes)
```

Comment:
:thinking: 
The purpose of doing so would be to avoid the usage of the DB on the `SyncRequestActor`s, so that it is only accessed by the `FastSync` actor, right? (which BTW is also used in `FastSyncReceiptsRequestHandler`)